stacks/upload-api-stack.js

```diff
@@ -1,60 +1,30 @@
 import {
   Api,
   Config,
-  Table,
   use
 } from '@serverless-stack/resources'
-import {
-  CarparkStack
-} from './carpark-stack.js'
+import { UploadDbStack } from './upload-db-stack.js'
+import { CarparkStack } from './carpark-stack.js'
 
-import { storeTableProps, uploadTableProps } from '../api/tables/index.js'
-import { getConfig, getCustomDomain, getApiPackageJson, getGitInfo, setupSentry } from './config.js'
+import { getCustomDomain, getApiPackageJson, getGitInfo, setupSentry } from './config.js'
 
 /**
  * @param {import('@serverless-stack/resources').StackContext} properties
  */
-export function ApiStack({ stack, app }) {
+export function UploadApiStack({ stack, app }) {
   stack.setDefaultFunctionProps({
-    srcPath: 'api'
+    srcPath: 'upload-api'
   })
 
-  // @ts-expect-error "prod" | "dev" | "staging" only allowed for stage
-  const stackConfig = getConfig(stack.stage)
-
   // Setup app monitoring with Sentry
   setupSentry(app, stack)
 
-  // Get carpark reference
+  // Get references from dependent stacks
```

Comment:
my "dependents" are things that depend on me, so here "dependent" stacks means stacks that depend on this stack
```suggestion
  // Get references to constructs created in other stacks
```